_schema.graphql

```diff
@@ -11679,15 +11679,6 @@ type PublishViewingRoomPayload {
 }
 
 type Query {
-  # Lot standings for a user
-  _unused_auctionsLotStandingConnection(
```

Comment:
I don't know why this is getting deleted from the schema - in fact `auctionsLots` should be getting added here. See below.